### PR TITLE
Update small nits regarding local non virt

### PR DESCRIPTION
**Pull request by afazekas**
* playbook name should include .yaml suffix
* the home directory usage is assumed
* The default branch is main not master or ""

This PR has:
- [ ] Appropriate testing (molecule, python unit tests)
- [ ] Appropriate documentation (README in the role, main README is up-to-date)

### CODE REVIEW

File: ci_framework/roles/edpm_prepare/tasks/main.yml [hunk 1]
```diff
@@ -9,8 +9,8 @@
       OPENSTACK_IMG: {{ operators_build_output[cifmw_operator_build_meta_name].image_catalog }}
       {% endif %}
     cifmw_edpm_prepare_make_openstack_deploy_env: |
+      OPENSTACK_BRANCH: "main"
```
Comment (pablintino) [inline ci_framework/roles/edpm_prepare/tasks/main.yml:12]: Under wich circunstances are you hitting errors here. The main purpose of a "" here is to point the branch and HEAD to the one the git_src_dir points. This is usually used only if building an operator from a PR.

Comment (afazekas) [reply ci_framework/roles/edpm_prepare/tasks/main.yml:12]: cifmw_installyamls_repos: "{{ ansible_user_dir }}/install_yamls" might not be followed by some parts. I'll make cleaner setup.

~/src/github/openstack-k8s-operators/install_yamls was used from old times.

File: docs/source/Quickstart/04_non-virt.md [hunk 2]
```diff
@@ -10,6 +10,9 @@ In order to do so, some steps should be done beforehand.
 
 ### 1. Get the project and its dependencies
 ```Bash
+$ cd $HOME
+$ mkdir -p $HOME/src/github/openstack-k8s-operators/
+$ cd $HOME/src/github/openstack-k8s-operators/
```
Comment (afazekas) [inline docs/source/Quickstart/04_non-virt.md:15]: .com ?

File: ci_framework/roles/operator_deploy/README.md [hunk 3]
```diff
@@ -5,7 +5,7 @@ Deploy only selected operator(s) on OpenShift
 None
 
 ## Parameters
-* `cifmw_operator_deploy_basedir`: (String) Directory where we will have the RHOL/CRC binary and the configuration (e.g. `artifacts/.rhol_crc_pull_secret.txt`). Default to `cifmw_basedir` which defaults to `~/ci-framework`.
+* `cifmw_operator_deploy_basedir`: (String) Directory where we will have the RHOL/CRC binary and the configuration (e.g. `artifacts/.rhol_crc_pull_secret.txt`). Default to `cifmw_basedir` which defaults to `~/ci-framework-data`.
 * `cifmw_operator_deploy_installyamls`: (String) `install_yamls` root location. Defaults to `cifmw_installyamls_repos` which defaults to `../..`.
```
Comment (pablintino) [inline ci_framework/roles/operator_deploy/README.md:9]: Be aware that the PR that changes this path is still not merged.https://github.com/openstack-k8s-operators/ci-framework/pull/202

File: docs/source/Quickstart/04_non-virt.md [hunk 4]
```diff
@@ -10,6 +10,11 @@ In order to do so, some steps should be done beforehand.
 
 ### 1. Get the project and its dependencies
 ```Bash
+$ cd $HOME
+$ mkdir -p $HOME/src/github.com/openstack-k8s-operators/
```
Comment (pablintino) [inline docs/source/Quickstart/04_non-virt.md:14]: Why is this needed? Those are paths that we typically use in CI, but the shouldn't be necessary. Agree in the add an explicit `cd $HOME`

Comment (afazekas) [reply docs/source/Quickstart/04_non-virt.md:14]: Looks like the example config does not overrides the location everywhere.
ATM better to copy the ci paths, until a fixing change lands at least the doc sample would be WORKSFORME.

File: ci_framework/roles/operator_deploy/README.md (hunk 3 above)
Comment (afazekas) [reply ci_framework/roles/operator_deploy/README.md]: removed

File: ci_framework/roles/edpm_prepare/tasks/main.yml (hunk 1 above)
Comment (afazekas) [reply ci_framework/roles/edpm_prepare/tasks/main.yml]: Not needed at the end.

File: docs/source/Quickstart/04_non-virt.md (hunk 2 above)
Comment (afazekas) [reply docs/source/Quickstart/04_non-virt.md]: Looks like both version used.
Adding a link.

File: docs/source/Quickstart/04_non-virt.md (hunk 4 above)
Comment (afazekas) [reply docs/source/Quickstart/04_non-virt.md:14]: Maybe the without ".com" version should be the default .
checking ..

File: ci_framework/roles/operator_deploy/README.md (hunk 3 above)
Comment (afazekas) [reply ci_framework/roles/operator_deploy/README.md]: The example config has a cifmw_basedir: "{{ ansible_user_dir }}/ci-framework-data" .

Comment (afazekas) [reply ci_framework/roles/operator_deploy/README.md]: install_yamls/devsetup/ci-framework/ansible.cfg still pointed to the ~/ci-framework .
After more cleanups the seams to be the right role location in the 

roles_path = ~/ci-framework-data/artifacts/roles:./ci_framework/roles:/usr/share/ansible/roles:/etc/ansible/roles:~/.ansible/roles

File: docs/source/Quickstart/04_non-virt.md [hunk 5]
```diff
@@ -57,11 +58,26 @@ pre_infra:
...
+
+Get the pull secret from `https://cloud.redhat.com/openshift/create/local`
+and place it to `$HOME/pull-secret`.
+
+### 5. Verifiy ansible.cfg
+
+install_yamls/devsetup/ci-framework/ansible.cfg
+The role_path needs to include cifmw_basedir .
+example:
+`roles_path = ~/ci-framework-data/artifacts/roles:./ci_framework/roles:/usr/share/ansible/roles:/etc/ansible/roles:~/.ansible/roles`
```
Comment (cjeanner) [inline docs/source/Quickstart/04_non-virt.md:71]: this shouldn't be needed anymore once #202 is in

File: docs/source/Quickstart/04_non-virt.md [hunk 6]
```diff
@@ -10,6 +10,7 @@ In order to do so, some steps should be done beforehand.
 
 ### 1. Get the project and its dependencies
 ```Bash
+$ cd $HOME
```
Comment (cjeanner) [inline docs/source/Quickstart/04_non-virt.md:13]: actually we don't really have to be in $HOME - any location where the current user can write would be OK.

File: docs/source/Quickstart/04_non-virt.md [hunk 7]
```diff
@@ -57,11 +58,26 @@ pre_infra:
...
+and place it to `$HOME/pull-secret`.
+
+### 5. Verifiy ansible.cfg
+
+install_yamls/devsetup/ci-framework/ansible.cfg
+The role_path needs to include cifmw_basedir .
+example:
+`roles_path = ~/ci-framework-data/artifacts/roles:./ci_framework/roles:/usr/share/ansible/roles:/etc/ansible/roles:~/.ansible/roles`
+
+You might want to remove the ~/ci-framework-data  directory before rerun.
```
Comment (cjeanner) [inline docs/source/Quickstart/04_non-virt.md:73]: we don't want to remove it, but ensure it's present. The generated install_yamls_makes role is there.

**Review by cjeanner — CHANGES_REQUESTED** [review]

File: docs/source/Quickstart/04_non-virt.md (hunk 6 above)
Comment (cjeanner) [reply docs/source/Quickstart/04_non-virt.md:16]: Not really? though that's a detail. Linking to the couple of parameters we can set in order to "move" locations is possible - let's go with your PR, and we can iterate with proper links and references.

File: docs/source/Quickstart/04_non-virt.md (hunk 7 above)
Comment (afazekas) [reply docs/source/Quickstart/04_non-virt.md]: Moved to the cleanup section.
This was about full reset.

**Pull request merged**